### DISCUSSION TITLE
[WARNING] [coolquasar] Incorrect signature for provider name

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [DNSCrypt/dnscrypt-proxy#1829] [Q&A]
Hi
What is this **WARNING**?

`[2021-08-17 15:04:20] [WARNING] [coolquasar] Incorrect signature for provider name: [2.dnscrypt-cert.jit.ddns.net.]`

I receive this WARNING yesterday and today
What should I do ?

**Comment by MojtabaFaal** [comment]
@jedisct1

**Reply by jedisct1** [reply]
That resolver was removed from the list on August 19.
